src/notifications/builder.ts

```diff
@@ -299,7 +301,7 @@ export class NotificationBuilder {
     if (source.type === SourceType.Squad && addSquadAvatar) {
       newBuilder = newBuilder.avatarSource(source);
     }
-    if (post.type === PostType.Share) {
+    if (post.type === PostType.Share && sharedPost) {
```

Comment:
Nice this might actually catch some edge-case fails 😅

Reply:
Yep, hopefully catch a lot more with strict enabled 🙏